explainers/beforematch.md

```diff
@@ -288,6 +307,50 @@ scroll after adding an async step.
 5. **content-visibility: hidden-matchable -> visible**: find-in-page scrolls to
 to the revealed text.
 
+### Alternatives
+Given the purpose of displaying `content-visibility: hidden-matchable` text
+when it is searched for, there are a number of alternatives we have considered.
+
+#### Automatic Unlocking
+`content-visibility: hidden-matchable` text would be automatically changed to
+`content-visibility: visible` when the hidden text is searched for.
+##### Pros
+* Provides the desired behavior without the need for the web developer to do
+  anything besides use the `hidden-matchable` value.
+* Less complex than `beforematch` with regards to scrolling behavior.
```

Comment:
We need to elaborate on this: Specifically, since script does not run and the interaction is fully within browser control, we can guarantee that scrolling to the element is possible

Reply:
I reworded this bullet point. Look good?

Reply:
Yep, this looks good.